Parsers/ASimRegistryEvent/Parsers/imRegistry.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
```

Comment:
Please rename this file to imRegistryEvent.yaml

Reply:
done